packages/eslint-config-mc-app/package.json

```diff
@@ -24,7 +24,7 @@
     "babel-eslint": "10.x",
     "eslint": "^5.16.0 || ^6.1.0",
     "eslint-config-airbnb-base": "14.x",
-    "eslint-config-prettier": "3.x",
+    "eslint-config-prettier": "3.x || 6.x",
```

Comment:
errrrrr

Reply:
At least it spots maddening outdated things :D

Reply:
It is smart though. it is not breaking.